tests/testthat/test_xml_conv_utils.R

```diff
@@ -0,0 +1,95 @@
+# Test the CSV to XML conversion utilities
+
+context("Test CSV to XML pipeline utilities")
+
+test_that("default MI header exists in the package", {
+  conv_test <- create_xml("test.xml")
+
+  expect_true(file.exists(conv_test$mi_header))
+})
+
+test_that("bogus MI header causes error", {
+  skip("TODO: need to find a way to get messages from Java")
+  conv_test <- create_xml("test.xml", "bogus.txt")
+
+  expect_message(run_xml_conversion(conv_test), regex="java.io.FileNotFoundException: bogus.txt", all=F)
+})
+
+test_that("converting without adding data causes error", {
+  skip("TODO: need to find a way to get messages from Java")
+  conv_test <- create_xml("test.xml")
+
+  expect_message(run_xml_conversion(conv_test), regex="java.lang.NullPointerException", all=F)
+})
+
+test_that("converting bogus data causes error", {
+  skip("TODO: need to find a way to get messages from Java")
+  `%>%` <- magrittr::`%>%`
```

Comment:
Nope, I guess I was assuming the package wasn't going to be loaded.  Should I back all of those out?